docs/docs/processing-payments-with-square.md

```diff
@@ -0,0 +1,106 @@
+
+## Choosing the right product
+
+Square can handle most of your payment-related needs including accepting payments, managing product catalogs, and managing payroll. Begin by determining which of these products you want to incorporate into your business. This guide focuses on [accepting payments on your website](https://developer.squareup.com/docs/online-payment-options#square-payments-in-your-own-website) but you may wish to use their in-person or in-app options as well.
+
+You've got two options for integrating Square payments: redirecting to a hosted checkout page with the Checkout API _or_ taking payments from your Gatsby site using the `SqPaymentForm` library with the Payments API.
+
+### Redirecting to a hosted checkout page
+
+Redirecting to a Square-hosted page takes some of the pressure off since you don't need to build a checkout page. However, getting that functionality "for free" does come with some restrictions. You will not be able to customize the UI of the page users are sent to once they're ready to checkout. Square only recommends this option for situations where accepting payments on your own site isn't feasible.
+
+### Accepting Square payments
+
+Square recommends using the Payments API instead because it offers much greater flexibility. You can customize not only the look and feel of the checkout process but also the checkout process itself, such as customizing the payment methods available.
+
+This process is broken into two steps:
+
+1. Generate a single-use token called a nonce.
+2. Charge whatever payment source the user has provided (this could be a credit card, gift card, etc.) using the nonce.
+
+To add a Square payment form to your Gatsby site, you'll need to load the JavaScript that runs Square's payment form into the `<head>` of your website. You can do this by calling `setHeadComponents()` in `gatsby-ssr.js`:
+
+```js:title=gatsby-ssr.js
+import React from "react";
+
+export const onRenderBody = ({ setHeadComponents }) => {
+  setHeadComponents([
+    <script src='https://js.squareup.com/v2/paymentform'></script>
+  ]);
+};
+
+```
+
+You'll also need to create some variation of a `PaymentForm` component. Square maintains a few [payment form templates](https://github.com/square/connect-api-examples/tree/master/templates/web-ui/payment-form) you can base your component on. Try starting with the ["basic" JavaScript file](https://github.com/square/connect-api-examples/blob/master/templates/web-ui/payment-form/basic/sqpaymentform-basic.js). They also provide a [running example](https://codesandbox.io/s/4zjrv7kry9?from-embed) using their "basic-digital-wallet" template with React.
```

Comment:
This seemed to be where I went wrong. I pulled out the basic JS file, but missed the "base your component" on piece. Wondering if there is a way to rephrase this and make this more prominent.